packages/patternfly-4/react-core/src/components/DataList/DataListContent.js

```diff
@@ -30,10 +18,25 @@ DataListContent.propTypes = {
   '': PropTypes.any
 };
 
-DataListContent.defaultProps = {
+const defaultProps = {
   className: '',
   id: '',
   isHidden: false
 };
 
+const DataListContent = ({ className, children, id, isHidden, 'aria-label': ariaLabel, rowid, ...props }) => (
+  <section
+    id={id}
+    className={css(styles.dataListExpandableContent, className)}
+    hidden={isHidden}
+    aria-label={ariaLabel}
```

Comment:
This might be out of scope for this PR and belong in a separate issue...

The props documentation states:

Name | Type | Required | Default | Description
-- | -- | -- | -- | --
aria-label | string |   |   | Adds accessible text to the DataList toggle


But the description seems to describe the DataListToggle component. In this context, it's providing an accessible label for the expandable content section.

And when looking at the DataListToggle component the following are documented:

Name | Type | Required | Default | Description
-- | -- | -- | -- | --
aria-labelledby | string |   | '' | Adds accessible text to the DataList toggle
aria-label | string |   | 'Details' | Adds accessible text to the DataList toggle

But only `aria-label` is accepted as a prop. If I provide a value for `aria-labelledby` on the component, that value doesn't display in the html. 

Currently, `aria-labelledby` is defined on the html element only if the consumer doesn't define `aria-label`. If `aria-label` is defined, then the `aria-labelledby` attribute is removed from the html. This is as designed (e.g. either the consumer uses our default label, which we generate using `aria-labelledby`, or they explicitly define the full text string using `aria-label`).

Reply:
You are correct, it's ignored. The old logic is:

`aria-labelledby={ariaLabel !== "Details" ? null : `${rowid} ${id}`}`

I'm changing it to
```
if (ariaLabel === "Details") {
    ariaLabelledBy = `${rowid} ${id}`;
}
```
since it probably wasn't intended for the property to do nothing.

Reply:
Thanks! These updates work as expected except for the following...

When I define `aria-label` on the `DataListToggle`, in the PR preview I will see `aria-labelledby` included without a value.

In the current implementation, `aria-labelledby` is not included on the element at all. This is the expected behavior.

As for removing `aria-labelledby` from the documentation for `DataListToggle`, should I create a separate issue for that?